### ISSUE TITLE
Elasticsearch authentication?

### ISSUE THREAD

**Issue by kamsz** [magneticio/vamp#1041]

**Comment by olafmol** [comment]
Afaik ES doesnt support authentication out of the box. either you add a proxy in front or you use their shield xpack

**Comment by kamsz** [comment]
Yeah, I'm asking how I should provide authentication details to Elasticsearch for Vamp? I've ES instance that has authentication enabled through x-pack.

**Comment by olafmol** [comment]
Ok Understood. Vamp enterprise supports secure elastic communication. Are you looking for "plain" authentication headers or more advanced shield/x-pack integration and support?

**Comment by kamsz** [comment]
Just a plain HTTP Basic auth.
